src/jquery.addressfield.js

```diff
@@ -179,6 +182,19 @@
   };
 
   /**
+  * Popualtes country dropdown with list of countries from provided json file if it is empty
+  */
+  $.fn.addressfield.initCountries = function(countryMap, selector){
```

Comment:
Seems fine to me; a little detail on what's being passed in in a docblock may be helpful.